seaborn/utils.py

```diff
@@ -568,18 +549,7 @@ def get_view_interval(self):
 
 def get_color_cycle():
     """Return the list of colors in the current matplotlib color cycle."""
-    try:
-        cyl = mpl.rcParams['axes.prop_cycle']
-        try:
-            # matplotlib 1.5 verifies that axes.prop_cycle *is* a cycler
```

Comment:
Can you double check that this issue is addressed in later matplotlib versions? It is written ambiguously and I'm not sure off the top of my head whether matplotlib enforces the presence of `color` in the `axes.prop_cycle`.

Reply:
This is never validated and is actually as bug on master since the fallback rcparam ``axes.color_cycle`` has been deprecated a while ago. So in this sense this PR doesn't change current code state. I'll open a separate issue for this bug.

Reply:
OK good, I agree that's best handled independently.